core/lib/dal/src/applied_snapshot_status_dal.rs

```diff
@@ -0,0 +1,142 @@
+use zksync_types::{snapshots::AppliedSnapshotStatus, L1BatchNumber, MiniblockNumber};
+
+use crate::StorageProcessor;
+
+#[derive(Debug)]
+pub struct AppliedSnapshotStatusDal<'a, 'c> {
```

Comment:
Since the table is renamed, it makes sense to rename this to `SnapshotRecoveryDal`, its getter to `snapshot_recovery_dal()`, `AppliedSnapshotStatus` to `SnapshotRecoveryStatus` etc. Technically can be done in a follow-up PR, but it shouldn't be hard 🙃

Reply:
corrected